website/docs/artist_details_panel.md

```diff
@@ -0,0 +1,40 @@
+---
+id: artist_details_panel
+title: The Task Details Panel
+sidebar_label: Details Panel
```

Comment:
```suggestion
title: The Details Panel
```

The details panel is not just for the task but for all entities.

Reply:
Thank you. 
https://github.com/ynput/ayon-documentation/pull/287/commits/069be35feeac7fabad542741b1bfa6d7643d8195